doc/api/http.md

```diff
@@ -917,6 +917,66 @@ Example:
 var contentType = response.getHeader('content-type');
 ```
 
+### response.getHeaderNames()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Array}
+
+Returns an array containing the names of the current outgoing headers. Do not
+expect the names to be sorted in any particular manner. All header names are
+lowercase.
```

Comment:
But it *may* be sorted, so saying it *is* 'unsorted' could be lying if V8/ChakraCore/etc. has/changes different behavior ;-)

Perhaps just leaving out any wording about sorted/unsorted would be best?